### DISCUSSION TITLE
ScriptReference/EditorUtility.RequestScriptReload

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#608] [Notes]
# ScriptReference/EditorUtility.RequestScriptReload

https://docs.unity3d.com/ScriptReference/EditorUtility.RequestScriptReload.html

**Comment by electronics45** [comment]
If assembly reloading has been disabled with `EditorApplication.LockReloadAssemblies();`, then this method does not do anything until the assemblies are unlocked again, upon which the assemblies will immediately begin to reload.
Interestingly, if assemblies are locked, and unlocked in the same frame that this method is called, the reload will happen synchronously (showing the progress bar we're all used to).  [Tested in 2020.3]
